src/modules/uORB/uORBManager.cpp

```diff
@@ -87,14 +87,14 @@ int uORB::Manager::orb_exists(const struct orb_metadata *meta, int instance)
 #endif
 }
 
-orb_advert_t uORB::Manager::orb_advertise(const struct orb_metadata *meta, const void *data)
+orb_advert_t uORB::Manager::orb_advertise(const struct orb_metadata *meta, const void *data, int queue_size)
```

Comment:
You're right, very inconsistent. I think I switched halfway between and did not update everything.